documentation/source/newsfragments/1457.feature.rst

```diff
@@ -0,0 +1,4 @@
+Add ability to provide custom entry points for libraries in ``GPI_EXTRA``.
+Entry points can be defined after ``:`` in the format ``lib_name_X:A_entry_point,lib_name_Y:B_entry_point``.
```

Comment:
You probably don't need this line, the crossref should do it.

Reply:
Agreed, let's keep the newsfragment concise